test/images/serve_hostname/VERSION

```diff
@@ -0,0 +1 @@
+v1.7
```

Comment:
This is because the existing version is in this format - https://github.com/kubernetes/kubernetes/blob/master/test/images/serve_hostname/Makefile#L22,